### ISSUE TITLE
New whitespace characters trigger unnecessary autocompletion

### ISSUE THREAD

**Issue by tobz1000** [graphql/graphiql#2182]
## Actual Behavior
In some scenarios, generic suggestions are activated when entering whitespace. For example, this is just after pressing enter, with the cursor on line 19:
![image](https://user-images.githubusercontent.com/1471978/154806661-5325a4cd-95e5-4992-be97-0f88b3a6bf09.png)

This is especially annoying when adding multiple line breaks, because the first suggestion is inserted into the file.

The problem doesn't occur when adding a line break to the very end of the file.

Typing a space within a comment has the same issue:
![image](https://user-images.githubusercontent.com/1471978/154806681-50bc330d-3743-4060-836b-917a107ca8f2.png)

## Expected Behavior
Suggestions are only shown when pressing `ctrl`-`space`, or when an actual word is being typed, not whitespace.

## Specifications

  - GraphQL for VSCode Extension Version: 0.3.41
  - VSCode Version: 1.64.2
  - OS Name: Pop OS 21.10
  - graphql config filename and format example:

**account.graphqls**
```graphql
type UserUpload {
  totalEarnedCents: Int!
  image: Image!
}

enum UserUploadsSortTerm {
  UploadDate
  ViewCount
  LikeCount
}

extend type Query {
  account: AccountInfo
}
```

**Comment by acao** [comment]
ah, this is a side effect of the new trigger characters in the language server. we added `/n` for fields, but this is an unpleasant side effect :/

**Comment by lucasconstantino** [comment]
Ps.: I've temporarily hacked around this issue by manually modifying this extension output codebase.

1. Open file `~/.vscode/extensions/graphql.vscode-graphql-0.4.4/out/server/index.js`
2. Find `triggerCharacters` (only one occurence)
3. Remove the line break (the empty template string breaking the line) from the array
4. Enjoy

**Comment by acao** [comment]
This bug was resolved in 0.4.8!
